src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -511,6 +531,7 @@ public void WriteTo(ObjectWriter writer)
                 writer.WriteString(FilePath);
                 writer.WriteString(OutputFilePath);
                 writer.WriteString(OutputRefFilePath);
+                CompilationOutputFilePaths.WriteTo(writer);
```

Comment:
Do we have any tests for serialization? I don't remember if this means it now works for free or if we also have to update some checksumming stuff.

Reply:
We do, I have improved them a bit.